test/factories/school_factory.rb

```diff
@@ -1,11 +1,18 @@
 class Cranky::Factory
-  # Simple factory method to create a user instance, you would call this via crank(:user)
   def school
-    School.new do |u|
-      u.id  = [1,1,1,1].map!{|x| (0..9).to_a.sample}.join.to_i
-      u.name  =   "Antonia Pantoja Preparatory Academy: A College Boar..."        
-      u.code  = 'zx' + rand.to_s[2..4]             
-      u.active =  true
-    end
+    define name: "Queens Academy: College Prep",
```

Comment:
Yes, this creates a new object.  It's the syntax I found on the gem's README at https://github.com/ginty/cranky

FYI, both syntaxes create rather than initialize the object.  When you call it with `crank!(:user)` it creates it right there.  When you call it with `crank(:user)` it is "lazy loaded" and created when called.